packages/testharness/src/utils/createWebSpeechMock.js

```diff
@@ -0,0 +1,372 @@
+import EventTarget, { defineEventAttribute } from 'event-target-shim';
```

Comment:
Is this file being used anywhere?

Reply:
Yes, this is used on line 51 of the same file.

We don't need the ES5 version because this is test harness.